code/modules/admin/admin_verbs.dm

```diff
@@ -1045,3 +1046,21 @@ var/list/admin_verbs_mod = list(
 	message_admins("[key_name_admin(usr)] has stopped all media.", 1)
 
 	stop_all_media()
+
+/client/proc/launch_event()

```

Comment:
All I found were some individual buttons in the secrets menu (like "spawn carps", "launch meteors" etc)